src/simulation/power/simpleBattery/simpleBattery.rst

```diff
@@ -36,3 +36,8 @@ The next step is to attach one or more :ref:`PowerNodeUsageMsgPayload` instances
 
 
 For more information on how to set up and use this module, see the simple power system example :ref:`scenarioPowerDemo`.
+
+To simulate a battery capacity fault that reduces the actual storage capacity (without directly altering the stated capacity), users must set up the fault flag::
```

Comment:
This is not very clear what we are doing here.  Can you please be more explicit on what is done when a fault is modeled?